include/rewd_controllers/JointTrajectoryControllerBase.hpp

```diff
@@ -155,6 +155,8 @@ class JointTrajectoryControllerBase
   // 14.04.
   realtime_tools::RealtimeBox<TrajectoryContextPtr> mCurrentTrajectory;
   std::atomic_bool mCancelCurrentTrajectory;
+  std::atomic_bool mAbortCurrentTrajectory;
```

Comment:
Can we add docstring to explain the difference, i.e. `mCancelCurrentTrajectory` is set true when the caller cancels the trajectory, and `mAbort...` is when the controller aborts it?